docs/runtime/runconfig.md

```diff
@@ -313,7 +317,9 @@ This comprehensive example configures an agent with:
 
 ### Enabling Experimental CFC Support
```

Comment:
There is a note (admonition) higher on this page that states:

> The support_cfc feature is experimental and its API or behavior might change in future releases.

Do we want to replace that note with these new badges?